src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,36 @@
+package duke;
+
+/**
+ * DukeException represents the possible exceptions that might occur while using Duke.
+ */
+public class DukeException extends RuntimeException {
+    private final String message;
```

Comment:
I think it would be better if constant names were in all uppercase? [(Reference: Constant names must be all uppercase using underscore to separate words)](https://se-education.org/guides/conventions/java/index.html#naming)

```suggestion
    private final String MESSAGE;
```

Reply:
I agree with you!